### PR TITLE
Improved the README description

### PR DESCRIPTION
**Pull request by theSage21**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # bhai
 bhai bhai bhai
+
+
+This holds code for the bhai lang written by bhai.
```
Comment (libhide) [inline README.md:5]: @theSage21 this readme was not written by a bhai, it was written by a behen. could you fix that please?

Comment (theSage21) [reply README.md:5]: Sure thing.

Comment (theSage21) [reply README.md:5]:
```suggestion
This holds code for the bhai lang written by THE bhai.
```

Comment (libhide) [reply README.md:5]: oh? THE bhai? 😮

**Pull request merged**